armTemplates/azuredeploy-blobforwarder.json

```diff
@@ -0,0 +1,154 @@
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "newRelicLicenseKey": {
+      "type": "string",
+      "metadata": {
+        "description": "New Relic License Key"
+      }
+    },
+    "newRelicEndpoint": {
+      "type": "string",
+      "defaultValue": "https://log-api.newrelic.com/log/v1",
+      "metadata": {
+        "description": "The Logs API endpoint used to send your logs to. By default, it is https://log-api.newrelic.com/log/v1 if your account is in the United States (US) region. Otherwise, if you're in the European Union (EU) region, you should use https://log-api.eu.newrelic.com/log/v1"
+      }
+    },
+    "storageAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "Name of the Azure Storage account."
+      },
+      "minLength": 3,
+      "maxLength": 24
+    },
+    "storageAccountSku": {
+      "type": "string",
+      "defaultValue": "Standard_LRS",
+      "allowedValues": [
+        "Standard_LRS",
+        "Standard_GRS",
+        "Standard_RAGRS",
+        "Standard_ZRS",
+        "Premium_LRS",
+        "Premium_ZRS",
+        "Standard_GZRS",
+        "Standard_RAGZRS"
+      ],
+      "metadata": {
+        "description": "Sku on which to run the Azure Storage account."
+      }
+    },
+    "storageAccountKind": {
+      "type": "string",
+      "defaultValue": "StorageV2",
+      "allowedValues": [
+        "Storage",
+        "StorageV2",
+        "BlobStorage",
+        "FileStorage",
+        "BlockBlobStorage"
+      ],
+      "metadata": {
+        "description": "Indicates the type of storage account."
+      }
+    },
+    "storageAccountLocation": {
+      "type": "string",
+      "metadata": {
+        "description": "Location where the storage account reside. (eg. Central US)"
+      }
+    },
+    "storageAccountContainer": {
+      "type": "string",
+      "metadata": {
+        "description": "A container in the storage account"
+      },
+      "minLength": 3,
+      "maxLength": 63
+    }
+  },
+  "variables": {
+    "storageAccountId": "[resourceId('Microsoft.Storage/storageAccounts', parameters('storageAccountName'))]",
+    "blobForwarderFunctionArtifact": "https://github.com/newrelic/newrelic-azure-functions/releases/latest/download/BlobForwarder.zip",
+    "onePerResourceGroupAndStorageAccountAndContainer": "[uniqueString(resourceGroup().id, parameters('storageAccountName'), parameters('storageAccountContainer'))]",
+    "functionAppName": "[concat('nrlogs-blobforwarder-', variables('onePerResourceGroupAndStorageAccountAndContainer'))]"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Storage/storageAccounts",
+      "apiVersion": "2021-04-01",
+      "name": "[parameters('storageAccountName')]",
+      "location": "[parameters('storageAccountLocation')]",
+      "sku": {
+        "name": "[parameters('storageAccountSku')]"
+      },
+      "kind": "[parameters('storageAccountKind')]",
+      "properties": {
+        "accessTier": "Hot",
```

Comment:
I'm worried about cost of this storage, maybe we could add some rule to manage it?
https://docs.microsoft.com/en-us/azure/storage/blobs/lifecycle-management-overview

I'm okey also to provide this info in the doc, and let the customer to manage it modifying the template, up to you

Reply:
Yes, let's provide docs about it! 👍

Reply:
Agree